src/js/service/ImportService.js

```diff
@@ -0,0 +1,121 @@
+/* Image and Animation import service supporting the import dialogs,
+ * file dropper service, and headless import API. */
+(function () {
+  /**
+   * @param {!PiskelController} piskelController
+   * @param {!PreviewController} previewController
+   * @constructor
+   */
+  var ImportService = $.namespace('pskl.service').ImportService =
```

Comment:
For consistency I would keep using the var ns = $.namespace(...) as we do in other files. Unless of course there is a good reason to do it differently here.

Not against changing this convention but if we do, let's do it everywhere.